tools/VerboseMaker.js

```diff
@@ -10,12 +10,19 @@ exports.description = 'print out information about POMs and files visited';
 
 exports.args = [
   {
+    class: 'Boolean',
     name: 'showFiles',
     description: 'enable showing of processed files',
     value: false
   }
 ];
 
+const { processArgs } = require('./buildlib');
+
+exports.init = function() {
+  processArgs(X, exports.args);
```

Comment:
I wonder how this worked without it? I see, values are still set if supplied from the command-line, just the default value isn't set if they aren't specified, which doesn't matter in this case because the default is false, which would be the effective default anyway.